src/main/webapp/app/shared/image-cropper/component/image-cropper.component.ts

```diff
@@ -170,7 +173,12 @@ export class ImageCropperComponent implements OnChanges, OnInit {
     }
 
     private isValidImageChangedEvent(): boolean {
-        return this.imageChangedEvent?.target?.files?.length > 0;
+        if (!this.imageChangedEvent?.currentTarget) {
+            return false;
+        }
+        const element = this.imageChangedEvent.currentTarget as HTMLInputElement;
+
+        return !!element.files && element.files.length > 0;
```

Comment:
```suggestion
        return element.files?.length;
```
This should also work